source/uwp/winui3/Renderer/AdaptiveCardRenderer.vcxproj

```diff
@@ -124,6 +124,7 @@
   <ItemDefinitionGroup Condition="'$(Configuration)'=='Debug'">
     <ClCompile>
       <PreprocessorDefinitions>_DEBUG;%(PreprocessorDefinitions)</PreprocessorDefinitions>
+      <DebugInformationFormat Condition="'$(Configuration)|$(Platform)'=='Debug|x64'">ProgramDatabase</DebugInformationFormat>
```

Comment:
This change resolved an error that I was seeing, but can remove if we don't need it in main